### ISSUE TITLE
Error about missing file or directory

### ISSUE THREAD

**Issue by zmand97** [jpiringer/LinearPredictionCoder#1]
When I try compiling and running the Xcode project file, it results in this error:
"PBXCp /usr/local/lib/libsndfile.1.dylib build/Debug/libsndfile.1.dylib
.../Linear Production Coder/LinearPredictionCoder-master/Builds/MacOSX"
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -exclude Headers -exclude PrivateHeaders -exclude Modules -exclude *.tbd -resolve-src-symlinks /usr/local/lib/libsndfile.1.dylib /Users/guest2.0/Downloads/downloaded\ plugins\ for\ editing/Linear\ Production\ Coder/LinearPredictionCoder-master/Builds/MacOSX/build/Debug

error: /usr/local/lib/libsndfile.1.dylib: No such file or directory"

Is there a way I can get rid of this error?

**Comment by jpiringer** [comment]
you need to install libsndfile.
e.g. enter in the terminal:
brew install libsndfile

**Comment by zmand97** [comment]
I entered that in the terminal, but after entering it, it said: "command not found." I'm going through possible solutions from ([this site](https://osxdaily.com/2018/05/24/command-not-found-mac-terminal-error-fix/). Can more than one of these options or solutions work on a Mac?

One of the solutions says that the problem could be caused by missing system files that need to be restored. Is there supposed to be a libsndfile on all Macs, or is it downloaded?

I also searched for anything labeled "libsndfile" on my Mac. There is a folder labeled "libsndfile-1.0.28" that seems to have come with the downloaded Linear Production Coder folder, but I'm not sure why it's not being detected, despite being in the folder.
